phpunit.xml.dist

```diff
@@ -1,11 +1,20 @@
-<phpunit
-	bootstrap="vendor/autoload.php"
-	colors="true"
-	>
-	<testsuites>
-		<testsuite>
-			<directory prefix="spec-" suffix=".php">tests/</directory>
-			<directory prefix="test-" suffix=".php">tests/</directory>
-		</testsuite>
-	</testsuites>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/4.8/phpunit.xsd"
+		 bootstrap="vendor/autoload.php"
+		 backupGlobals="false"
+		 beStrictAboutCoversAnnotation="true"
+		 beStrictAboutOutputDuringTests="true"
+		 beStrictAboutTestsThatDoNotTestAnything="true"
+		 beStrictAboutTodoAnnotatedTests="true"
+		 colors="true"
+		 verbose="true">
```

Comment:
We might want to remove it from here, as PHPUnit doesn't support it in future releases. Verbisuty can be handled from CLI instead if required?